invenio_rdm_records/services/review/service.py

```diff
@@ -86,6 +96,16 @@ def create(self, identity, data, record, uow=None):
         # Set the request on the record and commit the record
         record.parent.review = request_item._request
         uow.register(RecordCommitOp(record.parent))
+
+        # TODO: remove before merge. dev only
```

Comment:
remove?

Reply:
It is only here for development purposes, to more easily trigger the whole workflow of the submission. Will remove before merge :)